src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Main program of the Duke chatbot. Consists of the
+ * user interface, current task list, task list saved
+ * on disk, and a parser of user input.
+ */
+public class Duke {
+    /** User interface that reads user input and prints to screen */
+    private Ui ui;
+
+    /** Parser to make sense of user input */
+    private Parser parser;
+
+    /** Current task list in the program */
+    private TaskList storedTasks;
+
+    /** Task list saved on disk */
+    private Storage savedTasks;
+
+    public Duke() {
+        ui = new Ui();
+        parser = new Parser();
+        storedTasks = new TaskList();
+        savedTasks = new Storage();
```

Comment:
I feel that consistency (there's an inconsistency in the use of `this.` in the Java files) is important in a code base, what do you think?